src/ResourceManager/KeyVault/Commands.KeyVault/Commands.KeyVault.csproj

```diff
@@ -49,7 +51,37 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="Commands\AddAzureKeyVaultCertificate.cs" />
+    <Compile Include="Commands\SetAzureKeyVaultCertificateAttribute.cs" />
+    <Compile Include="Commands\StopAzureKeyVaultCertificateOperation.cs" />
```

Comment:
Act of adding a certificate 'starts' a certificate operation. So there is no explicit pair for stop.
Polling happens using Get-AzureKeyVaultCertificateOperation and checking on the status field till it reaches and end state (completed/failed).